library/src/layout/table.rs

```diff
@@ -288,7 +288,8 @@ impl LocalName for TableElem {
         match lang {
             Lang::ARABIC => "جدول",
             Lang::BOKMÅL => "Tabell",
-            Lang::CHINESE => "表",
+            Lang::SIMP_CHINESE => "表",
+            Lang::TRAD_CHINESE => "表",
```

Comment:
simp has a few questionable implications in English.
```suggestion
            Lang::CHINESE_SIMPLIFIED => "表",
            Lang::CHINESE_TRADITIONAL => "表",
```